coreneuron/apps/corenrn_parameters.cpp

```diff
@@ -184,13 +184,21 @@ void corenrn_parameters::parse(int argc, char** argv) {
                   << std::endl;
         app.exit(e);
         throw e;
-
     } catch (const CLI::ParseError& e) {
         // use --help is also ParseError; in this case exit by showing all options
         app.exit(e);
         exit(0);
     }
 
+#ifndef CORENEURON_ENABLE_GPU
+    if (gpu) {
+        std::cerr << "GPU support was not enabled at build time but --gpu was passed. This is "
+                     "probably not what you meant."
```

Comment:
this could be simplified a bit:
```suggestion
        std::cerr << "Error: GPU support was not enabled at build time but GPU execution is requested."
```